src/renderer/components/+preferences/editor.tsx

```diff
@@ -69,6 +69,27 @@ export const Editor = observer(() => {
           onChange={value => editorConfiguration.tabSize = Number(value)}
         />
       </section>
+      <section>
+        <SubTitle title="Font size"/>
+        <Input
+          theme="round-black"
+          type="number"
+          min={10}
+          validators={InputValidators.isNumber}
+          value={editorConfiguration.fontSize.toString()}
+          onChange={value => editorConfiguration.fontSize = Number(value)}
```

Comment:
With `<input type="number">` using `Number()` probably redundant.

Reply:
Good idea, though I think we should instead start writing a new Input component from the ground up as the current one is very massive.

Reply:
This is even better. But probably out of scope of this PR

Reply:
Definitely